### PR TITLE
freshchat updated to allow styles custmization and aar building

### PR DESCRIPTION
**Pull request by CodeCombiner**
Added submodule of android freshchat sdk

### CODE REVIEW

**Comment by rampr** [comment]
Hi Alex, Please explain the changes here

**Comment by CodeCombiner** [comment]
Hi Ram

I updated Freshchat to make it's Android layouts to be customizable. It shell work like this:
1) install plugins
2) go to game/modules directory
3) Import project to Android Studio
4) change required Android layouts
5) Rebuild project (new .aar will be assembled)
6) build Android project or copy new aar to existing Android project

The plugin uses slightly modified fork of original Freshchat Android SDK as submodule.

Please let me know if this is not the best approach.

**Comment by rampr** [comment]
As discussed, we are not using this right now. We will come back to this when we need these changes in the future.
#14 is sufficient for our needs.

**Comment by rampr** [comment]
I was previously wrong. These changes are needed. Reopening it.

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "android/freshchat-android"]
+	path = android/freshchat-android
+	url = https://github.com/CodeCombiner/freshchat-android
```
Comment (bijoshtj) [inline .gitmodules:3]: should be change to hashcube/freshchat android

Comment (rampr) [reply .gitmodules:3]: We will change it after all changes are done.  Thanks for pointing it out.

**Pull request merged**